src/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Set default PHP version
-FROM php:8.2.15-apache
+FROM php:8.3.3-apache
```

Comment:
HumHub currently doesn't fully support PHP 8.3 so we need to wait till v1.16.x for more testing.